R/gen.R

```diff
@@ -0,0 +1,73 @@
+#' Create a function to draw from the posterior predictive distribution for a
+#' latent model
+#'
+#' This function creates a function that draws from the posterior predictive
+#' distribution for a latent model using [primarycensored::rpcens()] to handle
+#' censoring and truncation. The returned function takes a prep argument from
+#' `brms` and returns posterior predictions. This is used internally by
+#' [brms::posterior_predict()] to generate predictions for latent models.
+#'
+#' @inheritParams epidist_family_model
```

Comment:
The documentation here you get is:

> Output of a call to [brms::brmsfamily()](https://paulbuerkner.com/brms/reference/brmsfamily.html) with additional information as provided by [.add_dpar_info()](file:///Users/adamhowes/Downloads/website/reference/dot-add_dpar_info.html)

Is this the intention? Is the additional `.add_dpar_info()` needed?